Automattic/wp-calypso
# Site Profiler: Fix the highlighted text

## Proposed Changes

Update the highlighted part

## Why are these changes being made?
To match the design of v2

## Testing Instructions
* Go to `/site-profiler/:url` with a WP.com site. Ex: `/site-profiler/wordpress.com`
* Check if the hosting title is similar to the picture below

| Before  | After |
| ------------- | ------------- |
![CleanShot 2024-05-22 at 15 59 32@2x](https://github.com/Automattic/wp-calypso/assets/5039531/8f539958-7f3f-44aa-9015-5dff69d69011)|![CleanShot 2024-05-22 at 16 00 10@2x](https://github.com/Automattic/wp-calypso/assets/5039531/8d287ade-f7c2-4ed8-9eca-b505a5ff1dd1)|


## Pre-merge Checklist
